docker/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:22.04
```

Comment:
If adding docker script, it should be put under `contrib/`, not the root, because container is not a blessed way to build it.

Reply:
Done